libs/encryption-sdk/src/main/java/org/opensearch/encryption/dummy/DummyCryptoProvider.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.encryption.dummy;
+
+import org.opensearch.common.crypto.CryptoProvider;
+import org.opensearch.common.crypto.DecryptedRangedStreamProvider;
+import org.opensearch.common.crypto.EncryptedHeaderContentSupplier;
+import org.opensearch.common.crypto.MasterKeyProvider;
+import org.opensearch.common.io.InputStreamContainer;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import com.amazonaws.encryptionsdk.caching.CachingCryptoMaterialsManager;
+
+public class DummyCryptoProvider implements CryptoProvider {
```

Comment:
Why do we need this in libs?

Reply:
`CryptoManager` needs 1 implementation of `CryptoProvider` to unblock itself. This is only a temporary provider for now . Once we have a `RealCryptoProvider` we can remove it . I will add a ToDo for this .